contract_price_revision/models/account_analytic_invoice_line.py

```diff
@@ -1,39 +0,0 @@
-# Copyright 2019 Tecnativa <vicent.cubells@tecnativa.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from odoo import api, fields, models
-import odoo.addons.decimal_precision as dp
-
-
-class AccountAnalyticINvoiceLine(models.Model):
-    _inherit = "account.analytic.invoice.line"
-
-    date_start = fields.Date(
-        string='Start Date',
-    )
-    date_end = fields.Date(
-        string='End Date',
-    )
-    previous_revision_id = fields.Many2one(
```

Comment:
We should convert this field in new `predecessor_contract_line_id` field in a migration script. And what about the dates?

Reply:
Script created.
Date fields has the same name and the table was renamed keeping all fields and values.